mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py

```diff
@@ -0,0 +1,156 @@
+from datetime import datetime
+
```

Comment:
We need a little documentation or link to the documentation here. This doc should explain how to package the dag and yamls file and to deploy it to a composer.